client/app/src/api/concertApi.js

```diff
@@ -36,9 +36,11 @@ const connect = idToken => {
     console.log('MESSAGE', message.data);
     try {
       const { event, data } = JSON.parse(message.data);
-      if (event === 'EVENT_STAGE_CHANGED') {
-        console.log('emitting SHOW_COLOR_PICKER event');
-        events.emit('SHOW_COLOR_PICKER');
+      if (event === "CONNECTED") {
```

Comment:
If we're just going to mirror the event and data, why bother with the if statements?

Reply:
I had the idea in my head that naming the events emitted by the EventEmitter differently from the events emitted by the websocket would lead to less confusion and make debugging easier. But thinking about it now, I don't see how it's any better than mirroring the event and data. I think I'll go ahead and remove the if statements.